packages/backend/migrations/20240820101201_add_deleted_at_grants.js

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
nit: why not combine this migration with `drop_unique_grants`?

Reply:
Yup, makes sense, let me do that